DataLayer/LightsData/LightsData.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "I_LightsData.h"
+
+class LightsData : public I_LightsData
+{
+public:
+    LightsData();
+    virtual ~LightsData();
+
+    /* LightsData Gets */
+    unsigned char getLightStatus() const;
+
+    /* LightsData Sets */
+    void setLightStatus(const unsigned char&);
+
+private:
+    unsigned char lightStatus_;
```

Comment:
These shoudln't be bitflags now that they're in the data layer.  They should just be boolean variables for each field.